src/interactive_list.js

```diff
@@ -0,0 +1,26 @@
+import { setLocalStorage, addTask } from './addremove.js';
+
+let Task = JSON.parse(localStorage.getItem('Task')) || [];
+
+const editCheckBox = (ID, event) => {
+  Task.forEach((element) => {
+    if (element.index === ID) {
+      element.completed = event;
+    }
+    setLocalStorage(Task);
+  });
+};
```

Comment:
While you did implement the previous reviewer's requirement, there is now strange behavior. After I edit a task, the last letter of the task is not saved to the local storage. Please fix it.

<img width="1440" alt="Screen Shot 2022-07-14 at 16 39 56" src="https://user-images.githubusercontent.com/82426682/179090812-cca1f31d-c454-4d91-8f22-bce21084c467.png">